golem-service-base/src/db/sqlite.rs

```diff
@@ -358,7 +358,8 @@ pub async fn migrate(
     info!("DB migration: sqlite://{}", config.database);
     let mut conn = SqliteConnection::connect_with(&config.connect_options()).await?;
     let migrator = sqlx::migrate::Migrator::new(migrations).await?;
-    migrator.run(&mut conn).await?;
+    // See: https://github.com/launchbadge/sqlx/issues/954, Send required for golem-cli
+    futures::executor::block_on(migrator.run(&mut conn))?;
```

Comment:
Not a fan of blocking the async runtime thread, but as this is only done during startup probably fine

Reply:
I hope nobody is fan of that :) see the linked issue, reasoning is what you wrote